src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -21,11 +22,14 @@ namespace NHSD.BuyingCatalogue.Organisations.Api
 {
     public class Startup
     {
+        private readonly IWebHostEnvironment _enviroment;
```

Comment:
Typo: `_enviroment` should be `_environment`.

Reply:
👍